pkg/httputil/httputil.go

```diff
@@ -45,8 +45,8 @@ func NewClient(credential *security.Credential) (*Client, error) {
 	}, nil
 }
 
-// IsFiltered return true if the given feedState matches the whiteList.
-func IsFiltered(whiteList string, feedState model.FeedState) bool {
+// IsNeeded return true if the given feedState matches the whiteList.
+func IsNeeded(whiteList string, feedState model.FeedState) bool {
```

Comment:
The name is a bit confusing for me 🤔
Also is it possible to bind this method to FeedState struct instead of putting it in the httputil?

Reply:
sounds good